### PR TITLE
1.3.5

### PR DESCRIPTION
**Pull request by weijunyu**
### Changes

- #101 Masthead font-size scaling.
- Masthead link now darkens upon hover.
- sgds.js no longer initializes a sticky sidebar by default. This feature is not sgds's
  responsibility and its removal reduces the js library size greatly. Documentation for 
  this feature to follow.
- Default colours for `info` and `link` classes have been updated.
- Added more font-weight classes (e.g. `has-text-weight-extra-bold`).
- scss sources are now published on NPM. Documentation for usage to follow.
- Documentation improvements.

### Fixes

- Fixed navbar burger button functionality after new navbar class introduced.

### CODE REVIEW

File: collections/_standard/masthead.html [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
 order: 1
-js: true
 ---
 
 <h4 class="has-text-primary">
-    The Masthead is a standard component that has to be present in all pages of
-    '.gov.sg' websites.
+    All .gov.sg digital services shall implement
+    the Official Government Banner
+    (containing the Singapore Government
```
Comment (Sylphias) [inline collections/_standard/masthead.html]: Maybe reword to (Which must include the Singapore Government Online Logo)?

File: collections/_layouts/responsiveness.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+<h5 class="has-text-weight-semibold margin--bottom--sm">Different column sizes per breakpoint</h5>
+<p>
+    You can define a column size for each viewport size: <b>mobile, tablet, desktop, widescreen and fullHD</b>.
+</p>
+
+
+<table class="table is-bordered is-size-8">
+  <thead>
+    <tr>
+      <td>Class</td>
```
Comment (Sylphias) [inline collections/_layouts/responsiveness.html]: I suggest putting a more verbose instruction as to what values x can take to avoid potential confusion. For eg:
<td>Class (Replace x with a value between 1-12) </td>

File: collections/_standard/masthead.html (hunk 1 above)
Comment (rav3) [reply collections/_standard/masthead.html]: Think we should keep it since its taken from the DSS Standard 3.9
